lib/kafka-worker.rb

```diff
@@ -82,14 +106,19 @@ module Handler
     end
     
     class_methods do
-      def consumes(topic, start_from_beginning = false)
+      def consumes(topic, retry_interval = 60, start_from_beginning = false)
```

Comment:
I think it's be better to move all those out into separate methods, i.e. you'd call 

```
class HelloHandler
  consumes "hello"
  retry_interval 60
  start_from_beginning true
end
```